zerver/lib/onboarding.py

```diff
@@ -217,7 +216,6 @@ def send_welcome_bot_response(send_request: SendMessageRequest) -> None:
     internal_send_private_message(welcome_bot, send_request.message.sender, content)
 
 
-@transaction.atomic
```

Comment:
I think we can keep this, just mark it `@transaction.atomic(savepoint=False)`

Typo in your commit message, BTW, of calling this function `send_inital_realm_messages` (missing an `i`)

Reply:
It does mean that `send_initial_realm_messages`, if called from somewhere else, would always properly be in a transaction.  But you're right that this only has one callsite, so there's no advantage, really.

👍